config.json: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1.16k/1.16k [00:00<?, ?B/s]
C:\Users\bilal\Documents\VLM_Medical\.venv\lib\site-packages\huggingface_hub\file_download.py:143: UserWarning: `huggingface_hub` cache-system uses symlinks by default to efficiently store duplicated files but your machine does not support them in C:\Users\bilal\.cache\huggingface\hub\models--Qwen--Qwen-VL-Chat. Caching files will still work but in a degraded version that might require more space on your disk. This warning can be disabled by setting the `HF_HUB_DISABLE_SYMLINKS_WARNING` environment variable. For more details, see https://huggingface.co/docs/huggingface_hub/how-to-cache#limitations.
To support symlinks on Windows, you either need to activate Developer Mode or to run Python as an administrator. In order to activate developer mode, see this article: https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development
  warnings.warn(message)
configuration_qwen.py: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2.09k/2.09k [00:00<00:00, 2.47MB/s]
A new version of the following files was downloaded from https://huggingface.co/Qwen/Qwen-VL-Chat:
- configuration_qwen.py
. Make sure to double-check they do not contain any added malicious code. To avoid downloading new versions of the code file, you can pin a revision.
modeling_qwen.py: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 44.7k/44.7k [00:00<00:00, 11.4MB/s]
Encountered exception while importing einops: No module named 'einops'
Encountered exception while importing transformers_stream_generator: No module named 'transformers_stream_generator'
Traceback (most recent call last):
  File "C:\Users\bilal\Documents\VLM_Medical\train.py", line 68, in <module>
    model = AutoModelForCausalLM.from_pretrained(
  File "C:\Users\bilal\Documents\VLM_Medical\.venv\lib\site-packages\transformers\models\auto\auto_factory.py", line 558, in from_pretrained
    model_class = get_class_from_dynamic_module(
  File "C:\Users\bilal\Documents\VLM_Medical\.venv\lib\site-packages\transformers\dynamic_module_utils.py", line 569, in get_class_from_dynamic_module
    final_module = get_cached_module_file(
  File "C:\Users\bilal\Documents\VLM_Medical\.venv\lib\site-packages\transformers\dynamic_module_utils.py", line 393, in get_cached_module_file
    modules_needed = check_imports(resolved_module_file)
  File "C:\Users\bilal\Documents\VLM_Medical\.venv\lib\site-packages\transformers\dynamic_module_utils.py", line 225, in check_imports
    raise ImportError(
ImportError: This modeling file requires the following packages that were not found in your environment: einops, transformers_stream_generator. Run `pip install einops transformers_stream_generator`
